lib/chef/knife/bootstrap_windows_winrm.rb

```diff
@@ -91,8 +91,16 @@ def wait_for_remote_response(wait_max_minutes)
           return
         rescue
           retries_left -= 1
+          # $! will contain the current exception object
+          if $!.class.equal?(Errno::ECONNREFUSED)
```

Comment:
Shouldn't this be `rescue Errno::ECONFFREFUSED => e` to avoid explicitly checking the type of the exception instance?